File: Exemplos SmartPOS/App_IntentDigitalHUB_Java/Elgin_IntentDigitalHUB_SmartPOS_Java-font/app/src/main/java/com/elgin/java_intentdigitalhub_smartpos/IntentDigitalHubService/TERMICA/Commands/DefinePosicao.java
package com.elgin.java_intentdigitalhub_smartpos.IntentDigitalHubService.TERMICA.Commands;

import com.elgin.java_intentdigitalhub_smartpos.IntentDigitalHubService.TERMICA.TermicaCommand;
import com.google.gson.JsonObject;

public class DefinePosicao extends TermicaCommand {
    final private int posicao;

    public DefinePosicao(int posicao) {
        super("DefinePosicao");
        this.posicao = posicao;
    }

    @Override
    protected JsonObject functionParametersJson() {
        JsonObject functionParametersJson = new JsonObject();

        functionParametersJson.addProperty("posicao", this.posicao);

        return functionParametersJson;
    }
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_SmartPOS_Kotlin-font/app/src/main/java/com/elgin/kotlin_intent_digital_hub_smartpos/IntentDigitalHubService/TERMICA/Commands/DefinePosicao.kt
package com.elgin.kotlin_intent_digital_hub_smartpos.IntentDigitalHubService.TERMICA.Commands

import com.elgin.kotlin_intent_digital_hub_smartpos.IntentDigitalHubService.TERMICA.TermicaCommand
import com.google.gson.JsonObject

class DefinePosicao(private val posicao: Int) : TermicaCommand("DefinePosicao") {
    override fun functionParametersJson(): JsonObject {
        val functionParametersJson = JsonObject()
        functionParametersJson.addProperty("posicao", posicao)
        return functionParametersJson
    }
}